lib/cli-engine.js

```diff
@@ -550,18 +505,8 @@ CLIEngine.prototype = {
         loadedPlugins[pluginNameWithoutPrefix] = pluginobject;
     },
 
-    /**
-     * Resolves the patterns passed into executeOnFiles() into glob-based patterns
-     * for easier handling.
-     * @param {string[]} patterns The file patterns passed on the command line.
-     * @returns {string[]} The equivalent glob patterns.
-     */
     resolveFileGlobPatterns: function(patterns) {
```

Comment:
Don't think we need this anymore. Just change all of the invocations to `globUtil`